tests/src/test/scala/whisk/core/database/UserCommandTests.scala

```diff
@@ -62,6 +62,12 @@ class UserCommandTests extends FlatSpec with WhiskAdminCliTestBase {
     admin.executeCommand().futureValue.right.get shouldBe key.compact
   }
 
+  it should "create a user with default key" in {
```

Comment:
can you audit the existing wskadmin tests, are there any other tests we should add here?

Reply:
sure would review them